client/src/app/pages/dashboard/components/controls-in-progress/controls-in-progress.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="root">
```

Comment:
The name is really strange.

Reply:
Besides the title, this one also contains _add a task_ button. See the above response for why it is named _controls_